### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by kaozdl**
add contribution guidelines

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+- Proposing new features
+- Becoming a maintainer
+
+## We Develop with Github
+We use github to host code, to track issues and feature requests, as well as accept pull requests.
+
+## We Use [Github Flow](https://guides.github.com/introduction/flow/index.html), So All Code Changes Happen Through Pull Requests
+Pull requests are the best way to propose changes to the codebase (we use [Github Flow](https://guides.github.com/introduction/flow/index.html)). We actively welcome your pull requests:
+
+1. Fork the repo and create your branch from `master`.
```
Comment (brunomichetti) [inline CONTRIBUTING.md:16]: I would here to make sure that travis passes

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+## Write bug reports with detail, background, and sample code
+[This is an example](http://stackoverflow.com/q/12488905/180626) of a bug report.
+Here's [another example from Craig Hockenberry](http://www.openradar.me/11905408), an app developer whom I greatly respect.
+
+**Great Bug Reports** tend to have:
+
+- A quick summary and/or background
+- Steps to reproduce
+  - Be specific!
+  - Give sample code if you can. [My stackoverflow question](http://stackoverflow.com/q/12488905/180626) includes sample code that *anyone* with a base R setup can run to reproduce what I was seeing
```
Comment (brunomichetti) [inline CONTRIBUTING.md:38]: What does `base R setup` mean?

**Review by brunomichetti — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (kaozdl) [reply CONTRIBUTING.md:16]: that is stated in step 4

File: CONTRIBUTING.md (hunk 2 above)
Comment (kaozdl) [reply CONTRIBUTING.md]: The guy from the example is using R.

File: CONTRIBUTING.md (hunk 1 above)
Comment (brunomichetti) [reply CONTRIBUTING.md:16]: Yes but you can understand that like 'local tests passes' but travis uses several versions of python and django. I would put something like `Ensure that local and travis tests pass`.
WDYT?

Comment (kaozdl) [reply CONTRIBUTING.md:16]: Travis would run as part of the CI. and we will request changes for that anyway.
For me it's clear as is.

Comment (brunomichetti) [reply CONTRIBUTING.md:16]: OK if you think is clear let's leave it that way

**Review by nsantos16 — APPROVED** [review]
Awesomee! It would nice to add a code of conduct too(you can be based in this [md file](https://github.com/rootstrap/react-base/pull/206/files))

**Review by brunomichetti — APPROVED** [review]

**Pull request merged**